client/src/frontend/components/composed/EntityHero/parts/CalloutList/styles.js

```diff
@@ -0,0 +1,51 @@
+import styled from "@emotion/styled";
+import { respond } from "theme/styles/mixins";
+import { breakpoints } from "theme/styles/variables/media";
+import { transientOptions } from "helpers/emotionHelpers";
+import { SocialLinks } from "../Social/styles";
+
+const BREAKPOINT = breakpoints[60];
+const GUTTER = "20px";
+
+export const Wrapper = styled("div", transientOptions)`
+  ${({ $mobile }) =>
+    $mobile
+      ? `display: block; ${respond(`display: none;`, BREAKPOINT)}`
+      : `display: none; ${respond(`display: block; width: 100%;`, BREAKPOINT)}`}
+
+  * + & {
+    margin-block-start: ${GUTTER};
+  }
+
+  + ${SocialLinks} {
+    margin-block: 35px;
+  }
+`;
+
+export const List = styled("div", transientOptions)`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+
+  > *:not(:first-child) {
+    width: 100%;
+    margin-block-start: ${GUTTER};
+  }
+
+  & + & {
+    margin-block-start: ${GUTTER};
+  }
```

Comment:
This space looks larger in the comps.

<img width="254" alt="Screen Shot 2021-12-07 at 3 11 53 PM" src="https://user-images.githubusercontent.com/12454203/145120464-e73846ca-829c-4bce-8597-ca2fdf1f1c4b.png">